tests/countries/test_ghana.py

```diff
@@ -0,0 +1,106 @@
+
+
+class TestGhana(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(Ghana, years=range(1957, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(Ghana, GH, GHA)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Ghana(years=1956))
+
+    def test_new_year_day(self):
+        self.assertHoliday(f"{year}-01-01" for year in range(1957, 2050))
+
+    def test_constitution_day(self):
+        self.assertHoliday(f"{year}-01-07" for year in range(2019, 2050))
+        self.assertNoHoliday(f"{year}-01-07" for year in range(1957, 2019))
+        self.assertNoHolidayName("Constitution Day", range(1957, 2019))
+
+    def test_founders_day(self):
+        self.assertHoliday(f"{year}-08-04" for year in range(2019, 2050))
+
+    def test_independence_day(self):
+        self.assertHoliday(f"{year}-03-06" for year in range(1957, 2050))
+
+    def test_farmers_day(self):  # 1st Friday of December
+        self.assertHolidayName(
+            "Farmer's Day",
+            "2023-12-01",
+            "2024-12-06",
+            "2025-12-05",
+            "2026-12-04",
+            "2027-12-03",
+            "2028-12-01",
+            "2029-12-07",
+            "2030-12-06",
+        )
+
+    def test_eid_ul_fitr(self):
+        self.assertHolidayName(
+            "Eid ul-Fitr (estimated)",
+            "2020-05-24",
+            "2021-05-13",
+            "2022-05-02",
+            "2023-04-21",
+            "2024-04-10",
+        )
+
+    def test_2023(self):
+        self.assertHolidays(
+            Ghana(years=2023),
+            ("2023-01-01", "New Year's Day"),
+            ("2023-01-02", "New Year's Day (observed)"),
+            ("2023-01-07", "Constitution Day"),
+            ("2023-01-09", "Constitution Day (observed)"),
+            ("2023-03-06", "Independence Day"),
+            ("2023-04-07", "Good Friday"),
+            ("2023-04-10", "Easter Monday"),
+            ("2023-05-01", "May Day"),
+            ("2023-04-21", "Eid ul-Fitr (estimated)"),
+            ("2023-06-28", "Eid ul-Adha (estimated)"),
+            ("2023-08-04", "Founders' Day"),
+            ("2023-09-21", "Kwame Nkrumah Memorial Day"),
+            ("2023-12-01", "Farmer's Day"),
+            ("2023-12-25", "Christmas Day"),
+            ("2023-12-26", "Boxing Day"),
+        )
+
+    def test_2024(self):
+        self.assertHolidays(
+            Ghana(years=2024),
+            ("2024-01-01", "New Year's Day"),
+            ("2024-01-07", "Constitution Day"),
+            ("2024-01-08", "Constitution Day (observed)"),
+            ("2024-03-06", "Independence Day"),
+            ("2024-03-29", "Good Friday"),
+            ("2024-04-01", "Easter Monday"),
+            ("2024-05-01", "May Day"),
+            ("2024-04-10", "Eid ul-Fitr (estimated)"),
+            ("2024-06-16", "Eid ul-Adha (estimated)"),
+            ("2024-06-17", "Eid ul-Adha (estimated) (observed)"),
```

Comment:
We've recently changed this a bit. So now it'd make sense to add `estimated_label` and `observed_estimated_label` (see, for example, [Brunei](https://github.com/JerryAgbesi/python-holidays/blob/c1d71cd6b72924d2e4984335e4abb1d71211a399/holidays/countries/brunei.py#L75)). Please note, you don't have to add l10n wrapper `tr(` unless you want/able to implement Akan l10n (that would be super cool for this or any other PR).

Reply:
Implementing Akan translations would be quite difficult but I'll check it out a raise an issue if necessary.

Reply:
> Implementing Akan translations would be quite difficult but I'll check it out a raise an issue if necessary.

That's completely optional. Thanks for considering this!